shell/platform/linux/fl_renderer_wayland.cc

```diff
@@ -9,6 +9,7 @@
 
 #include <gdk/gdkwayland.h>
 #include <wayland-egl-core.h>
+#include <cstring>
```

Comment:
Move this up to the top of the group alphabetically per the [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes). Other than that, LGTM!

Reply:
Actually super-quick for me just to patch it inline. Thanks again for the fix!

Reply:
>Move this up to the top of the group alphabetically per the style guide. 

This will break the CI.
Here is the suggestion from bot:
```
-#include <cstring>
 #include <gdk/gdkwayland.h>
 #include <wayland-egl-core.h>
+#include <cstring>
 ```

Should I push my original version?

Reply:
>Move this up to the top of the group alphabetically per the style guide.

As your change broke CI I re-uploaded my original version.
Based on my experience it's more important to adhere to what `clang-format` says then [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes). Looks like they are out of sync and `clang-format` takes precedence.

Reply:
Odd -- will take a look what's up. Thanks for re-uploading the fixed version.